src/client/pydaos/pydaos_core.py

```diff
@@ -148,6 +148,46 @@ def __repr__(self):
         return 'daos://{}/{}'.format(self.pool, self.cont)
 
 
+class DCheck():
+    """
+    Class representing of DAOS python container checker
+    Can be identified via a path or a combination of pool label and container
+    label (alternatively, UUID strings are supported too).
```

Comment:
So this class exists only to check the existence of a container? Why not use a function instead of a class?

Reply:
i was existing that we want to add the corruption test tool here, but i think im just going to create a separate tool for that. so i can covert that to a function.. does it make a difference?

Reply:
I see you changed to a function - thanks.